docs/advanced.rst

```diff
@@ -279,3 +279,32 @@ Or you can pass the change reason explicitly:
     poll = Poll(question='Question 1')
     poll.save()
     update_change_reason(poll, 'Add a question')
+
+Save Without Historical Record
+------------------------------
+
+You want save model without saving a historical record. You can use this method. 
+
+.. code-block:: python
+    def save_without_historical_record(self, *args, **kwargs):
```

Comment:
I think the lack of line break here might cause a rendering issue. I haven't tried to build this yet though.